scripts/templates/init-defaultworker.template

```diff
@@ -8,7 +8,16 @@ userdel hydro-service \
  && chown -R hydro-service:storage-hydro /hydroshare /tmp /shared_tmp /tmp \
  && chmod -R 3777 /tmp /shared_tmp /tmp 
 
+# we should be wary of setting concurrency on k8s
+# as it may not be the best way to scale celery workers
+# https://docs.celeryq.dev/en/v5.2.7/userguide/workers.html#autoscaling
+# autoscale values are hard coded here but should be tuned for production
+# based on number of nodes and the cpu limits set in the helm chart
+# https://github.com/CUAHSI/platform-recipes/blob/main/hydroshare/chart/values.yaml#L49-L56
+
 celery -A hydroshare beat  -s /hydroshare/celery/celerybeat-schedule &
-celery -A hydroshare worker -E -Q default --concurrency=CELERY_CONCURRENCY &
+
+# Start worker with autoscale and two queues
+celery -A hydroshare worker -E -Q default,periodic --autoscale=7,1 &
```

Comment:
The separation of periodic tasks makes a lot of sense to me, keeps the operational async tasks separate.

Reply:
I don't see or understand the separation between the queues as far as the processes. Is it possible for the periodic queue to use all 7 processes, leaving the default queue with nothing? The way this is written it looks like both queues share the processes.

Reply:
Good point thanks Scott! I separated it into two swparate workers